Dockerfile

```diff
@@ -16,9 +16,7 @@ COPY ./poetry.lock ./pyproject.toml ./
 
 # Install python dependencies using poetry
 RUN poetry config virtualenvs.create false
-RUN poetry export --with dev > requirements.txt
-RUN pip3 install --no-cache -r requirements.txt
-RUN pip3 install --no-cache "git+https://github.com/facebookresearch/detectron2.git@v0.5#egg=detectron2"
+RUN poetry install
```

Comment:
this might be what's in `make install`

Reply:
i seem to remember @joel-wright having issues with the size of the dockerfile spiralling but this is very much second hand news so feel free to ignore if you know otherwise

Reply:
https://pip.pypa.io/en/stable/topics/caching/

This might make sense actually, it might actually mean in this instance "don't store the http response for any future pip installs"

Reply:
![image](https://github.com/climatepolicyradar/navigator-document-parser/assets/58440325/bc9406e9-b686-4163-b2f4-fe82b71840ff)

Reply:
Let's just leave as is for now, the docker image size will already be way smaller